platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -24,7 +24,7 @@ NS_ASSUME_NONNULL_BEGIN
  alternatively observe KVO change notifications to the pack’s `progress` key
  path.
  */
-extern NSString * const MGLOfflinePackProgressChangedNotification;
+extern NSNotificationName const MGLOfflinePackProgressChangedNotification;
```

Comment:
Same comment here. MGLOfflinePackProgressChangedNotification namespaces it so I don't think we need the word `Notification` in the name. Also, along the lines of `swift_name`, does this change things enough for some bleeding edge users that we would want to hold off until a larger semver change?